doc/api/crypto.md

```diff
@@ -2422,6 +2422,25 @@ const ciphers = crypto.getCiphers();
 console.log(ciphers); // ['aes-128-cbc', 'aes-128-ccm', ...]
 ```
 
+### `crypto.getCipherInfo(nameOrNid)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `nameOrNid`: {string|number} The name or nid of the cipher to query.
+* Returns: {Object}
+  * `name` {string} The name of the cipher
+  * `nid` {number} The nid of the cipher
+  * `blockSize` {number} The block size of the cipher.
+  * `ivLength` {number} The expected or default initialization vector length or
+    zero if the cipher does not use an initialization vector.
```

Comment:
Quoting myself from https://github.com/nodejs/node/pull/26612#issuecomment-476741364:

> If there is a single valid IV length, you are right, it should be determined automatically. But as I said before, sometimes, there is no single correct choice, many ciphers accept multiple IV lengths. In that case, I'd probably ask the user for the IV size; if they know enough about crypto to pick a cipher for a specific application, then they should also be able to supply its parameters. What would you do? For example, what should the IV size for AES-CCM be? pycryptodome, a popular crypto library, says 11 bytes, a reasonably strong choice. OpenSSL only uses 7 bytes, a much weaker choice, which has the advantage of permitting larger messages to be encrypted. Similar problems arise with authentication tag lengths etc. We would have to decide without knowing what the user is going to do with the cipher.

I'd suggest to rename this property to make it clear that this is not necessarily the only valid IV length, but can also be an arbitrary default value chosen by OpenSSL that conflicts with the default values used by other crypto implementations. Blindly relying on this value will cause interoperability problems, e.g., if an application uses the default value provided by this API, it will not be able to communicate with applications that use crypto libraries that use other "default" values.

Reply:
I've clarified this in the documentation. Hopefully better